src/utils/assets/formatAssetFromServer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Normalizes the asset object returned from the API server
+ *
+ * @param {Object} input
+ * @param {string} input.asset_type_id UUID
+ * @param {string} input.created_at ISO 8601 Extended Format date/time string
+ * @param {string} input.description
+ * @param {string} input.id UUID
+ * @param {string} input.label
+ * @param {string} input.organization_id UUID
+ * @param {string} input.updated_at ISO 8601 Extended Format date/time string
```

Comment:
Looking at this, I realize I need to tighten up the stuff I did last week in #41 in this regard. Which of these is optional?

Reply:
`input.description` is optional. Updating 👍